### ISSUE TITLE
Synchronous XHR warning

### ISSUE THREAD

**Issue by danvk** [evanw/node-source-map-support#49]
When I source `browser-source-map-support.js` in Chrome 42, I get the following warning in the JS console:

> Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check http://xhr.spec.whatwg.org/.

**Comment by evanw** [comment]
Yup, this sucks. Logged a bug against the spec: https://www.w3.org/Bugs/Public/show_bug.cgi?id=28505. We'll see how it goes.

**Comment by sjungwirth** [comment]
I was hoping to use this with Zombie headless browser but that doesn't support synchronous XMLHttpRequests: https://github.com/assaf/zombie/issues/417

**Comment by DominicTobias** [comment]
This should be easy to fix - https://github.com/evanw/node-source-map-support/blob/master/source-map-support.js#L99

Either have an `install` and an `installAsync` - or the inverse to be more like Node (but more of a compatibility break)

**Comment by wbt** [comment]
@DominicTobias Were you able to get a fix working?
